src/path.ts

```diff
@@ -0,0 +1,12 @@
+import * as path from 'path';
```

Comment:
For the tests, could we perhaps run tests twice, once with path mocked/set to `path.posix` and once with `path.win32`? https://nodejs.org/api/path.html#path_windows_vs_posix

Reply:
no. Just don't use `path`